.circleci/config.yml

```diff
@@ -237,3 +397,40 @@ workflows:
             - rspec-tests
             - cucumber-tests
             - other-tests
+      - build-and-push-docker:
+          requires:
+            - upload-coverage
+      - hold-dev:
```

Comment:
**not a blocker**. could filter to not require dev deploy approval for master branch